### PR TITLE
Validator for currency

### PR DESCRIPTION
**Pull request by JoaoFGuiomar**
Added isValidCurrency validator

### CODE REVIEW

**Comment by JoaoFGuiomar** [comment]
#6

File: src/validators/__tests__/isValidCurrency.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import isValidCurrency from 'validators/isValidCurrency'
+
+test('Returns false if currency is invalid', () => {
+  expect(isValidCurrency(undefined)).toBe(false)
+  expect(isValidCurrency(1234)).toBe(false)
+  expect(isValidCurrency('1.123,50323 €')).toBe(false)
+  expect(isValidCurrency('1,000,00 €')).toBe(false)
+  expect(isValidCurrency('1000€')).toBe(false)
```
Comment (hugo-fonseca) [inline src/validators/__tests__/isValidCurrency.test.js]: @JoaoFGuiomar i think we should allow this one

Comment (JoaoFGuiomar) [reply src/validators/__tests__/isValidCurrency.test.js]: [Source](http://publications.europa.eu/code/pt/pt-370303.htm)
screenshot

Comment (hugo-fonseca) [reply src/validators/__tests__/isValidCurrency.test.js]: @JoaoFGuiomar well can't argue with that 😂

**Comment by hugo-fonseca** [comment]
@JoaoFGuiomar gj mate 🎉

**Pull request merged**